<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{background-color:black;color-white;}
QAction{background-color:white;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="rectangle_label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:white}</string>
    </property>
    <property name="text">
     <string>Now press the location of the final coordinate of the rectangle</string>
    </property>
   </widget>
   <widget class="QLabel" name="rectangle_label_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:white}</string>
    </property>
    <property name="text">
     <string>Press the location of one of the corners of the rectangle</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <widget class="QMenu" name="add_horizontal_menu">
     <property name="title">
      <string>Add Bar</string>
     </property>
     <addaction name="add_horizontal_action"/>
     <addaction name="add_vertical_action"/>
    </widget>
    <addaction name="add_horizontal_menu"/>
    <addaction name="add_stage_rectangle_action"/>
   </widget>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="open_location_action"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="change_bar_size_action"/>
   </widget>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="save_stage_layout_action"/>
   </widget>
   <addaction name="menuAdd"/>
   <addaction name="menuOpen"/>
   <addaction name="menuEdit"/>
   <addaction name="menuNew"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="add_stage_rectangle_action">
   <property name="text">
    <string>Add rectangle square</string>
   </property>
  </action>
  <action name="open_location_action">
   <property name="text">
    <string>Open Stage Layout</string>
   </property>
  </action>
  <action name="change_bar_size_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Change Bar Size</string>
   </property>
  </action>
  <action name="save_stage_layout_action">
   <property name="text">
    <string>Save stage layout</string>
   </property>
  </action>
  <action name="add_horizontal_action">
   <property name="text">
    <string>Add Horizontal Bar</string>
   </property>
  </action>
  <action name="add_vertical_action">
   <property name="text">
    <string>Add Vertical Bar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
